dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+      "displayName": {
+        "en": "Temperature"
+      },
+      "name": "temperature",
+      "schema": "double",
+      "unit": "degreeCelsius"
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration X/Y/Z components [mg]",
+      "displayName": {
+        "en": "Acc Value [mg]"
+      },
+      "name": "accelerometer",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single acceleration sample. I.e.: {a_x, a_y, a_z}."
+        },
+        "displayName": {
+          "en": "Single acceleration sample."
+        },
+        "fields": [
+          {
+            "name": "a_x",
+            "schema": "double"
+          },
+          {
+            "name": "a_y",
+            "schema": "double"
+          },
+          {
+            "name": "a_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Gyroscope X/Y/Z components [dps]",
+      "displayName": {
+        "en": "Gyro Value [dps]"
+      },
+      "name": "gyroscope",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single angular velocity sample. I.e.: {g_x, g_y, g_z}."
+        },
+        "displayName": {
+          "en": "Single angular velocity sample."
+        },
+        "fields": [
+          {
+            "name": "g_x",
+            "schema": "double"
+          },
+          {
+            "name": "g_y",
+            "schema": "double"
+          },
+          {
+            "name": "g_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration Peak Status X/Y/Z components [m/sˆ2]",
+      "displayName": {
+        "en": "Acc Peak Status [m/sˆ2]"
+      },
+      "name": "predictiveaccelerationstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, accpeak_x, accpeak_y, accpeak_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Acceleration Peak Status sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak X:"
+            },
+            "name": "accpeak_x",
```

Comment:
Perhaps capital P here and below, as in "accPeak"

Reply:
[335ab63](https://github.com/Azure/iot-plugandplay-models/pull/449/commits/335ab63428b102711a254df3e3b7f64320e9601a)